test/unit/spec/OktaSignIn_spec.js

```diff
@@ -640,13 +644,20 @@ Expect.describe('OktaSignIn v1 pipeline bootstrap ', function () {
 
 Expect.describe('OktaSignIn v2 bootstrap', function () {
   let signIn;
+  let codeVerifier;
+  let codeChallenge;
+  let codeChallengeMethod;
```

Comment:
nit: Consider making a global `context` object that has these keys to avoid managing multiple variables.